lib/active_model_serializers/railtie.rb

```diff
@@ -1,7 +1,17 @@
 require 'rails/railtie'
 
-module ActiveModel
+module ActiveModelSerializers
   class Railtie < Rails::Railtie
+    initializer 'active_model_serializers.action_controller' do
+      ActiveSupport.on_load(:action_controller) do
+        ActiveSupport.run_load_hooks(:active_model_serializers, ActiveModelSerializers)
+        include ::ActionController::Serialization
+        ActionDispatch::Reloader.to_prepare do
+          ActiveModel::Serializer.serializers_cache.clear
+        end
+      end
+    end
```

Comment:
@dgynn awesome! Thanks for looking at this! Feel free to submit a PR into it if you'd like.

Also, see https://github.com/rails-api/active_model_serializers/issues/1353 which is similar to https://github.com/rails-api/active_model_serializers/pull/1295